README.md

```diff
@@ -1,11 +1,25 @@
-# Speakerline
+<!-- # Speakerline -->
 
 [![Build Status](https://travis-ci.org/nodunayo/speakerline.svg?branch=master)](https://travis-ci.org/nodunayo/speakerline)
 [![Dependency Status](https://gemnasium.com/badges/github.com/nodunayo/speakerline.svg)](https://gemnasium.com/github.com/nodunayo/speakerline)
 
 > Showcasing speakers' proposals and timelines in a effort to demystify the CFP process and help new speakers get started
 
-## Where did the idea come from?
+# Contents
+
+* [Where did the idea come from?](#where-did-the-idea-come-from)
+* [Goals of Speakerline](#goals-of-speakerline)
+* [Using the speakerline website](#using-the-speakerline-website)
+* [Setting up for local development](#setting-up-for-local-development)
+  * [Getting started with Ruby and Postgres](#get-started-Ruby-Postgres)
+  * [Getting the Rails app running](#getting-rails-app-running)
+    * [Environment variables](#environment-variables)
+  * [Running the tests](#running-the-tests)
+* [Contributing to Speakerline](#contributing-to-speakerline)
```

Comment:
"Contributing to Speakerline" is in the same section as "Setting up for local development", so this needs indenting in by one.